Makefile

```diff
@@ -35,7 +37,23 @@ IMG_BASE ?= quay.io/sustainable_computing_io
 OPERATOR_IMG ?= $(IMG_BASE)/kepler-operator:$(VERSION)
 
 .PHONY: all
-all: build
+all: ## default target - setup a k8s cluster with operator installed
```

Comment:
cluster-up doesn't take into account the env variable that we set [here](https://github.com/sustainable-computing-io/kepler-operator/blob/464c5221d96bc765833371b8e7510429889791bf/Makefile#L80). It brings up the cluster with default values.

Reply:
is the default value fine? Note, this is for a "default" make target - all